makefile

```
@@ -0,0 +1,5 @@
+all::
+	PWD := $(shell pwd)
+	
+guides:
```

```suggestion
html-docker:
```

Maybe this should have HTML in the name since it only builds that? Should we also have a way to build the PDF in Docker in the future?

---

> Maybe this should have HTML in the name since it only builds that? Should we also have a way to build the PDF in Docker in the future?

How about `docs-html` `docs-pdf` `docs-epub` `docs-all`